### DISCUSSION TITLE
how to install plugins ?

### DISCUSSION THREAD

**Discussion by strawberrymiku** [stakira/OpenUtau#1070] [Q&A] [answered]
first time Ever using utau, trying to use english kasane teto and u need to use that plugin for them apparently .. i fear i have No idea how to install plugins other than drag n dropping them , which adds them as singers which i presume i dont want ?

thank you in advance !!

**Comment by oxygen-dioxide** [comment, marked as answer]
You don't need a plugin to use teto english voicebank. Just use the builtin en xsampa phonemizer and you can input English words

**Reply by strawberrymiku** [reply]
okay , thank you !!! :3
